types/rlp_unmarshal_storage.go

```diff
@@ -67,46 +104,113 @@ func (t *Transaction) UnmarshalStoreRLPFrom(p *fastrlp.Parser, v *fastrlp.Value)
 		return err
 	}
 
-	if len(elems) != 2 {
-		return fmt.Errorf("expected 2 elements")
+	if len(elems) != 2 && len(elems) != 3 {
+		return fmt.Errorf("expected 2 or 3 elements")
```

Comment:
Nitpick, but this should be a standardized error with `errors.New`

Reply:
Fixed at 7f41c6ec7c5545c7d76b828445ba99a2cfa3f1f7